examples/c/xmake.lua

```diff
@@ -0,0 +1,60 @@
+add_rules("mode.release", "mode.debug")
+add_rules("platform.linux.bpf")
+set_license("GPL-2.0")
+
+add_requires("libelf", "zlib")
+add_requires("linux-tools", {configs = {bpftool = true}})
+if is_plat("android") then
+    add_requires("ndk >=22.x", "argp-standalone")
+    set_toolchains("@ndk", {sdkver = "23"})
+else
+    add_requires("llvm >=10.x")
+    set_toolchains("@llvm")
+    add_requires("linux-headers")
+end
+
+add_includedirs("../../vmlinux")
+
+target("libbpf")
```

Comment:
Also, probably add a comment above this line saying something like "we use the vendored libbpf sources for libbpf-bootstrap. for some projects you may want to use the system-installed libbpf"

Reply:
Ok, I added it. and add an option `xmake f --system-libbpf=y`